### PR TITLE
Update libnl3 spec file to skip tests

### PR DESCRIPTION
**Pull request by lisibann**

### CODE REVIEW

File: libnl3/centOS/7/libnl3.spec [hunk 1]
```diff
@@ -1,3 +1,8 @@
+# The tests are disabled by default.
+# Set --with tests or bcond_without to run tests.
+# Original behaviour is preserved.
```
Comment (alanoe) [inline libnl3/centOS/7/libnl3.spec]: I'm wondering if this line adds anything useful. What exactly "original behaviour is preserved" means here?

Comment (lisibann) [inline libnl3/centOS/7/libnl3.spec]: In fact, I've just used the same lines added to the others spec files. It means that there was no changes in the %check stage.

Comment (alanoe) [inline libnl3/centOS/7/libnl3.spec]: Could you add a patch to this pull request to replace this "original behaviour is preserved" sentence with the clearer description you just provided?

**Comment by olavphibm** [comment]
start tests

**Comment by lisibann** [comment]
start tests

File: libnl3/centOS/7/libnl3.spec (hunk 1 above)
Comment (lisibann) [inline libnl3/centOS/7/libnl3.spec]: Yes. Doing that.

File: docker/centOS/7/docker.spec [hunk 2]
```diff
@@ -1,6 +1,6 @@
 # The tests are disabled by default.
 # Set --with tests or bcond_without to run tests.
-# Original behaviour is preserved.
+# Original behaviour is preserved: there was no changes in the %check stage.
```
Comment (olavphibm) [inline docker/centOS/7/docker.spec]: I don't think this line is necessary. We are not saying where we have modified the spec file and it is not even obvious where we got the file from (this is something we should improve), so saying the %check stage was not changed does not mean much. Anyway, the patch is fine for me.

Comment (alanoe) [inline docker/centOS/7/docker.spec]: Makes sense. What we should do is keep track of the changes between our spec and base distro spec. I'm ok with removing the "Original behaviour is preserved" sentence in all specs.

Comment (lisibann) [inline docker/centOS/7/docker.spec]: Ok. I'll remove this line.

**Pull request merged**